jbpm-workitems/jbpm-workitems-email/src/test/java/org/jbpm/process/workitem/email/SendHtmlTest.java

```diff
@@ -254,6 +256,16 @@ public void useEmailWorkItemHandlerWithAuthentication() throws Exception {
 
         for (WiserMessage wiserMessage : messages ) {
             MimeMessage message = wiserMessage.getMimeMessage();
+            String content = ((String) message.getContent()).replace("\r\n", "");
+            assertEquals(workItem.getParameter("To"), 
+              ((InternetAddress)message.getRecipients(RecipientType.TO)[0]).getAddress());
+            assertEquals(workItem.getParameter("From"),
+                        ((InternetAddress)message.getFrom()[0]).getAddress());          
+            assertEquals(workItem.getParameter("DisplayName"), 
+            		     ((InternetAddress)message.getFrom()[0]).getPersonal());
+            assertEquals(workItem.getParameter("Reply-To"),
+            		    ((InternetAddress)message.getReplyTo()[0]).getAddress() );         
+            assertEquals( workItem.getParameter("Body"), content );
```

Comment:
Please just double check if your formatting settings are correct, since there is a space before and after the parenthesis. Also `(InternetAddress)message` should be `(InternetAddress) message`.

Reply:
Thanks for reviewing, fixed!